website/docs/developer-docs/setup/full-dev-environment.mdx

```diff
@@ -13,133 +13,184 @@ import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
 import TabItem from "@theme/TabItem";
 import Tabs from "@theme/Tabs";
 
-## Requirements
+## Prerequisites
+
+Before you begin, ensure you have the following tools installed:
 
 - [Python](https://www.python.org/) (3.13 or later)
-- [uv](https://docs.astral.sh/uv/getting-started/installation/), (Latest stable release)
+- [uv](https://docs.astral.sh/uv/getting-started/installation/) (Latest stable release)
 - [Go](https://go.dev/) (1.24 or later)
 - [Node.js](https://nodejs.org/en) (24 or later)
 - [PostgreSQL](https://www.postgresql.org/) (16 or later)
-- [Redis](https://redis.io/) (7 or later)
 - [Docker](https://www.docker.com/) (Latest Community Edition or Docker Desktop)
 - [Docker Compose](https://docs.docker.com/compose/) (Compose v2)
 
-## Services Setup
+## 1. Setting Up Required Services
+
+authentik depends on several external services:
+
+- [Redis](https://redis.io/) for caching
+- [PostgreSQL](https://www.postgresql.org/) for database storage
+- [Zenko CloudServer (S3)](https://www.zenko.io/cloudserver/) for object storage
+- [Sentry Spotlight](https://spotlightjs.com/) for error tracking and visualization
+
+### Option A: Using Docker Compose (Recommended)
+
+The easiest way to set up these services is using the provided Docker Compose configuration:
+
+```shell
+docker compose -f scripts/docker-compose.yml up -d
+```
+
+### Option B: Using local installations
 
-For PostgreSQL and Redis, you can use the `docker-compose.yml` file in `/scripts`. To use these pre-configured database instances, navigate to the `/scripts` directory in your local copy of the authentik git repo, and start the services by running `docker compose up -d`.
-Alternatively, you can also use a native install, if you prefer.
+Alternatively, you can install and run these services directly on your system.
 
 :::info
-If you use locally installed databases, the PostgreSQL credentials given to authentik should have permissions for `CREATE DATABASE` and `DROP DATABASE`, because authentik creates a temporary database for tests.
+If using locally installed databases, ensure the PostgreSQL credentials provided to authentik have `CREATE DATABASE` and `DROP DATABASE` permissions, as authentik creates temporary databases for testing.
```

Comment:
```suggestion
If using locally installed databases, ensure the PostgreSQL credentials provided to authentik have `CREATE DATABASE` and `DROP DATABASE` permissions, because authentik creates temporary databases for testing.
```

Reply:
The rest of the rewording is great, but for sure we want to use "because" instead of "as" (because "as" has many meanings, and is less ESL-friendly).